### PR TITLE
errors_only flag implementation; added steps for invalid docker submissions

### PR DESCRIPTION
**Pull request by vpchung**

### CODE REVIEW

**Review by thomasyu888 — CHANGES_REQUESTED** [review]
Looks good, but you have to make `docker_repository` and `docker_digest` optional parameters in the CWL.

**Comment by vpchung** [comment]
Oh right, forgot to make that change there too.

Thanks!

**Review by thomasyu888 — CHANGES_REQUESTED** [review]
I think the argparse in the inline python has to be optional too?

**Comment by vpchung** [comment]
Ah yes, you're right! Should've just copied-pasted .___."

Thanks!

File: validate_email.cwl [hunk 1]
```diff
@@ -56,25 +61,27 @@ requirements:
...
           evaluation = syn.getEvaluation(sub.evaluationId)
+          message = subject = ""
           if args.status == "INVALID":
             subject = "Submission to '%s' invalid!" % evaluation.name
             message = ["Hello %s,\n\n" % syn.getUserProfile(userid)['userName'],
                        "Your submission (%s) is invalid, below are the invalid reasons:\n\n" % sub.name,
                        args.invalid,
                        "\n\nSincerely,\nChallenge Administrator"]
-          else:
+          elif not args.errors_only:
```
Comment (thomasyu888) [inline validate_email.cwl:71]: Kenny brought this up with me before, and this is just being knit-picky now, but and `if-else` clause should be confined to the same logic.   So:

logically speaking`if args.status == "INVALID"`, the `elif` or `else` would pertain to the `args.status`.    I understand this `elif` logic though, just FYI for the future.

**Review by thomasyu888 — APPROVED** [review]

File: validate_email.cwl (hunk 1 above)
Comment (vpchung) [reply validate_email.cwl:71]: Oh interesting.  Okay, I'll keep that in mind for next time!

Comment (thomasyu888) [reply validate_email.cwl:71]: Ya it just makes your code more intuitive to other people that read it. Thanks for the contribution!

**Pull request merged**